quarkchain/cluster/jsonrpc.py

```diff
@@ -839,6 +839,7 @@ def get_data_default(key, decoder, default=None):
                     "blockHeight": quantity_encoder(tx.block_height),
                     "timestamp": quantity_encoder(tx.timestamp),
                     "success": tx.success,
+                    "is_coinbase": tx.is_coinbase,
```

Comment:
May use the name "is_from_root_chain"?  Since in the future, we plan to support tx in root_chain and allow root_chain to send a transaction to different shards.

Reply:
sounds good. will update the name